### PR TITLE
[argtable3] Add new port

### PR DESCRIPTION
**Pull request by myd7349**
https://github.com/argtable/argtable3
Fix https://github.com/argtable/argtable3/issues/32

### CODE REVIEW

**Comment by myd7349** [comment]
https://github.com/argtable/argtable3/pull/38

**Comment by myd7349** [comment]
argtable/argtable3#38

File: ports/argtable3/fix-cmake.patch [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+ endif()
+@@ -57,8 +68,18 @@ target_include_directories(argtable3 PRIVATE ${PROJECT_SOURCE_DIR}/src)
+ set_target_properties(argtable3 argtable3_static PROPERTIES
+   VERSION ${PROJECT_VERSION_MAJOR}.${PROJECT_VERSION_MINOR}.${PROJECT_VERSION_PATCH}
+   SOVERSION ${PROJECT_VERSION_MAJOR}.${PROJECT_VERSION_MINOR})
+-
+-install(TARGETS argtable3 EXPORT ${ARGTABLE3_PACKAGE_NAME}Config DESTINATION ${ARGTABLE3_INSTALL_LIBDIR})
+-install(TARGETS argtable3_static EXPORT ${ARGTABLE3_PACKAGE_NAME}Config ARCHIVE DESTINATION ${ARGTABLE3_INSTALL_LIBDIR})
+-install(FILES "${PROJECT_SOURCE_DIR}/src/argtable3.h" DESTINATION ${ARGTABLE3_INSTALL_INCLUDEDIR})
++if(BUILD_SHARED_LIBS)
```
Comment (vicroms) [inline ports/argtable3/fix-cmake.patch]: We should install just one target `argtable3` and use a `vcpkg-cmake-wrapper.cmake` to provide the other one.

Here's an example in `czmq`.

https://github.com/microsoft/vcpkg/blob/b8f0e2c8c69dc1990e175b1eecfb3a40db71a46a/ports/czmq/vcpkg-cmake-wrapper.cmake#L1-L9

**Review by vicroms — CHANGES_REQUESTED** [review]

File: ports/argtable3/fix-cmake.patch (hunk 1 above)
Comment (myd7349) [reply ports/argtable3/fix-cmake.patch]: Hi! @vicroms Thanks!

**Pull request merged**